include/deal.II/fe/fe_enriched.h

```diff
@@ -644,6 +648,160 @@ class FE_Enriched : public FiniteElement<dim, spacedim>
                           internal::FEValuesImplementation::FiniteElementRelatedData<dim,spacedim>    &output_data) const;
 };
 
+
+
+namespace ColorEnriched
+{
+  /**
+   * An alias template for predicate function which returns a
+   * boolean for a Triangulation<dim>::cell_iterator object.
+   *
+   * This is used by helper functions and in the implementation of
+   * ColorEnriched::Helper class.
+   */
+  template <int dim>
+  using predicate_function
+    = std::function< bool
+      (const typename Triangulation<dim>::cell_iterator &) >;
+
+  /**
+   * ColorEnriched::Helper class provided to create hp::FECollection, a
```

Comment:
class creates a collection of finite elements (hp::FECollection) to be used with hp::DoFHandler...